examples/bot.py

```diff
@@ -15,7 +15,7 @@
 # With our client established, let's have the library inform us when the client is ready.
 # These are known as event listeners. An event listener can be established in one of two ways.
 # You can provide the name of the event, prefixed by an "on_", or by telling the event decorator what event it is.
-@client.event
+@client.event(name="on_ready")
```

Comment:
You don't need name if the function name is the events name

Reply:
Removed the name param